sequoia/settings/offline_rl/setting.py

```diff
@@ -0,0 +1,40 @@
+import d3rlpy
+from sklearn.model_selection import train_test_split
+from typing import ClassVar, List, Tuple, Dict
+
+
+from sequoia.settings.base import Setting
+from torch.utils.data import DataLoader
+from sequoia.settings.base import Method
+from dataclasses import dataclass
+
+from simple_parsing.helpers import choice
+
+
+@dataclass
+class OfflineRLSetting(Setting):
+    available_datasets: ClassVar[list] = ["cartpole-replay",  # Cartpole Replay
+                                          "cartpole-random",  # Cartpole Random
+                                          ]
+    dataset: str = choice(available_datasets, default="cartpole-replay")
+    create_mask: bool = False
+    mask_size: int = 1
```

Comment:
Can you add some comments above/below these to give an idea of what they do?
BTW, the comments will also become the description for each argument from the command-line when someone uses `--help`. (e.g. `sequoia run offline_rl --help`)

Reply:
Is there a specific format for adding my comments? Or are we adding the --help info manually?

Reply:
Nah regular comments are fine.

Reply:
Can you add a description for these? what do they do?

Reply:
One small thing: move the comments next to (above) the respective fields.